rules/falco_rules.yaml

```diff
@@ -2698,7 +2698,7 @@
     Remote file copy tool launched in container (user=%user.name user_loginuid=%user.loginuid command=%proc.cmdline parent_process=%proc.pname
     container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
   priority: NOTICE
-  tags: [network, process, mitre_lateral_movement, mitre_exfiltration]
+  tags: [network, process, mitre_TA0008_lateral_movement, mitre_TA0010_exfiltration, mitre_TA0006_credential_access, mitre_T1552_unsecured_credentials]
```

Comment:
the detection scenarios are not for `mitre_TA0006_credential_access` and `mitre_T1552_unsecured_credentials`